admin_guide/out_of_resource_handling.adoc

```diff
@@ -12,31 +12,47 @@ toc::[]
 
 == Overview
 
-The node must preserve node stability when available compute resources are
-low. This is especially important when dealing with incompressible resources
-such as memory or disk. If either resource is exhausted, the node becomes
-unstable.
+The node must preserve node stability when available compute resources are low.
+This is especially important when dealing with incompressible resources such as
+memory or disk. If either resource is exhausted, the node becomes unstable.
+
+[WARNING]
+====
+Failure to disable swap memory makes the node not recognize it is under
+*MemoryPressure*.
+
+To take advantage of memory based evictions, operators must
+xref:../admin_guide/overcommit.adoc#disabling-swap-memory[disable swap].
+====
 
 [[out-of-resource-eviction-policy]]
 == Eviction Policy
 
-The node can proactively monitor for and prevent against total starvation of a
-compute resource. In cases where this appears to occur, the node can
-proactively fail one or more pods in order to reclaim the starved resource.
-When the node fails a pod, it terminates all containers in the pod, and the
-`*PodPhase*` is transitioned to *Failed*.
+Using eviction policies, a node can proactively monitor for and prevent
+against total starvation of a compute resource.
+
+In cases where a node is running low on available resources, it can proactively
+fail one or more pods in order to reclaim the starved resource using an eviction
+policy. When the node fails a pod, it terminates all containers in the pod, and
+the `*PodPhase*` is transitioned to *Failed*.
```

Comment:
With the updated doc guidelines, we can apply the new formatting here:

    `PodPhase`